### PR TITLE
Add chain id parameter to MSRC

### PR DESCRIPTION
**Pull request by palango**
As discussed before.

### CODE REVIEW

File: src/raiden_libs/cli.py [hunk 1]
```diff
@@ -100,10 +100,15 @@ def call_with_common_options_initialized(**params: Any) -> Callable:
             try:
                 setup_logging(params.pop("log_level"))
                 if not params["state_db"]:
-                    chain_id = str(params["web3"].net.version)
+                    # only RC has `chain_id`, MS and PFS have `web3` object
+                    chain_id = str(params.get("chain_id") or params["web3"].net.version)
```
Comment (karlb) [inline src/raiden_libs/cli.py]:
```suggestion
                    chain_id = str(params.pop("chain_id", params["web3"].net.version))
```

Then you should not need the extra pop below.

File: src/request_collector/cli.py [hunk 2]
```diff
@@ -2,22 +2,49 @@
...
+from raiden.utils.cli import NetworkChoiceType
 from raiden_libs.cli import common_options
 
 log = structlog.get_logger(__name__)
 
 
 @click.command()
+@click.option(
+    "--chain-id",
+    type=NetworkChoiceType(["mainnet", "ropsten", "rinkeby", "goerli", "kovan", "<NETWORK_ID>"]),
```
Comment (karlb) [inline src/request_collector/cli.py]: Nice! Is there a way to reuse the helpstring, too?

File: src/raiden_libs/cli.py (hunk 1 above)
Comment (palango) [reply src/raiden_libs/cli.py]: Nice, I expected this but didn't find it in any of the first google results.

File: src/request_collector/cli.py (hunk 2 above)
Comment (palango) [reply src/request_collector/cli.py]: Don't think so, but we can make it a constant and use that in the future.

File: src/raiden_libs/cli.py (hunk 1 above)
Comment (palango) [reply src/raiden_libs/cli.py]: So actually this doesn't work, as it eagerly evaluated and would complain about `web`  not being a valid key.

Comment (karlb) [reply src/raiden_libs/cli.py]: Well, then you could still do
```python
chain_id = str(params.pop("chain_id", None) or params["web3"].net.version)
```

Comment (palango) [reply src/raiden_libs/cli.py]: Yes, but as we need to delete it as well when the `state-db` is given, I think it's cleaner that way.

**Pull request merged**